File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        // if(n<0){
        //     return 0;
        // }
        // if(n<2){
        //     return 1;
        // }
        // return climbStairs(n-1) + climbStairs(n-2);
        int a =1;
        int b = 1;
        for(int i=2;i<=n;i++){
            int c= a+b;
            a = b;
            b = c;
        }
        return b;
    }
}

File: 0070-climbing-stairs/0070-climbing-stairs.js
/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
    let a=1,b = 1
    for(let i=2;i<=n;i++){
        let c = a+b;
        a=b
        b=c
    }
    return b
};